### DISCUSSION TITLE
Couldn't load object in AR

### DISCUSSION THREAD

**Discussion by AytoMaximo** [google/model-viewer#2213] [Q&A]
### Description

One our 3D model has some unknown issues preventing its loading in AR (scene-viewer). We've validated this model with the help of https://arvr.google.com/scene-viewer-preview, but there are no errors, just a few warnings. We've almost tried to reduce model size (<10mb), remove textures and reduce number of triangles (<100k), even remove animation. No luck. 

Also, I have such a problem on my Google Pixel XL, however Pixel 3 works fine. 

![image](https://user-images.githubusercontent.com/14392087/89521026-6deeb880-d7e7-11ea-9aee-9e2b8b4097c8.png)

#### Live Demo
https://rosatom-bdcd9.firebaseapp.com/#Flot

### Version
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
<script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>

### Browser Affected

- [x] Chrome

### OS

- [x] Android

**Comment by AytoMaximo** [comment]
An important update. It's not an issue about 3d model at all. I've found out that for some reason scene-viewer tries to load an old file, however in 3D mode there is a new one. 

What've I've done: I took another 3D model with another id and links, changed source glb on the file server and opened with my phone (of course after clean cache). I saw my ship in 3D mode, however proceeding to AR it opened an old model.

From what source scene-viewer take a file for AR if cleaning cache and rebooting devices don't help with it? 

A whole day was lost and we were about to go crazy :D

**Comment by elalish** [comment]
I have no idea how you got an out-of-date model URI in scene-viewer when it was up-to-date in model-viewer. @tpsiaki any ideas? You shouldn't have to worry about this in webXR mode, since that does not require downloading the model again.

**Comment by AytoMaximo** [comment]
I'm using iFrame, so no WebXR for me nowadays

**Comment by tpsiaki** [comment]
@AytoMaximo Scene Viewer is distributed in the Google app so it's possible that the Http Cache in this app could have an out-of-date version of the model.  In the future if you run into this you can clear the Google app cache and that would force the model to reload.

**Comment by AytoMaximo** [comment]
@tpsiaki Thank you for an answer! Just to be sure, are you talking about this guy?

![Screenshot_20200812-192924](https://user-images.githubusercontent.com/14392087/90041808-7a38b100-dcd2-11ea-9743-a9ebe7e5e265.png)

**Comment by tpsiaki** [comment]
Yeah that's the one.

**Comment by chandanwebkul** [comment]
Hi am reopen this issue
Facing same issue with following details
I am using version 3.1.4.
It's working at my developer instance but at production instance it's not working.
Showing couldn't load object at android device
it's working on IOS, but issue with android device
with chrome broswer.
I have checked cloudflare, i am not using cloudflare.

<model-viewer style="margin: 0px auto; display: block;" data-js-focus-visible="" src="https://yodacart.com/shop/pub/media/catalog/product/glbmodels/TVLG006.glb" alt="A 3D model of an product" auto-rotate="" ar="" camera-controls="" touch-action="pan-y" shadow-intensity="1" camera-orbit="33deg 67deg auto" ar_preferred="" ar-modes="scene-viewer webxr quick-look" ar-status="not-presenting">
         <button id="ar-button" slot="ar-button" style="background-color: transparent;border-radius: 25px;
         border: 1px solid darkgray;position: absolute;bottom: 0px;right: 26%;">
            View In Your Space
        </button>
    </model-viewer>
```javascript
 setTimeout(() => {
            var link = document.getElementById('ar-button');
            link.click();
            console.log("click evnet occur");
        }, 3000)
